vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Installing and Using Packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Installing and Using Packages on Linux
+
+## Description
+
+In this tutorial, you will learn how to use vcpkg to set up a new "Hello World" project using the `fmt` library, manage dependencies, build, and run your C++ application. 
+
+## Setup vcpkg
+
+1. **Clone the vcpkg repository**
+
+   Downloads the vcpkg registry.
+
+    ```bash
+    git clone https://github.com/microsoft/vcpkg.git
+    ```
+
+   vcpkg's curated registry refers to a set of libraries that have been vetted and packaged to work smoothly with the vcpkg package management system. These libraries are usually open-source projects from various domains, and they are included in vcpkg's main GitHub repository. Each package or "port" in the curated registry typically contains:
+
+   **a.** `Portfile.cmake`: A CMake script that contains instructions for downloading, building, and installing the library.
+
+   **b.** `vcpkg.json`: Information like version numbers, dependencies, and other data that vcpkg uses to manage the package.
+
+   **c.** Patches: Any necessary patches to make the library work with vcpkg or to fix issues in the library itself.
```

Comment:
```suggestion
   **c.** Patches: Any necessary patches to make the library work with vcpkg or to fix issues in the library itself.
```

I think there's no need to mention patches just yet. Or at least remark that patches aren't always needed.